main.go

```diff
@@ -40,6 +40,11 @@ func main() {
 		log.Fatalf("failed to create app: %v", err)
 	}
 
+	err = app.SetupLogger()
+	if err != nil {
+		log.Fatalf("failed to setup logger: %v", err)
```

Comment:
This error looks non-critical - the app can probably continue serving its main purpose despite not writing logs to disk. Let's just `log.Printf` instead.